canister/src/types.rs

```diff
@@ -54,10 +54,34 @@ impl Default for Config {
 
 #[derive(CandidType, Serialize, Deserialize, PartialEq, Eq, Debug, Clone, Default)]
 pub struct Fees {
-    pub get_utxos: u128,
+    /// The base fee to charge for all `get_utxos` requests.
+    pub get_utxos_base: u128,
```

Comment:
Note that these changes will require us to temporarily add some `serde[default]`s. I'll be adding those, along with hard-coding the actual fees we'll use for the launch, in a separate commit.

Reply:
What about the `get_utxos` field you're removing? Isn't that a potential issue with candid?

Reply:
It does break the `set_config` API, but that's a private endpoint so that isn't an issue.

Reply:
I'm also not entirely sure that decoding an old state with the new struct would work. You tested specifically the upgrade of the bitcoin canister to this version on the testnet?